static/templates/move_topic_to_stream.hbs

```diff
@@ -0,0 +1,7 @@
+<ul class="nav nav-list topic-move_popover">
+    {{#each streams}}
+    <li>
+        <a href="#" class="popover-move-topic-to-stream" data-stream-id="{{ this.stream_id }}" data-message-id="{{ @root.message_id }}">{{this.name}}</a>
```

Comment:
The `data-message-id` part could be put on the `ul` element rather than duplicating it on each stream.

Reply:
Also, let's prefix the stream names with a `#`.